tests/admin_views/tests.py

```diff
@@ -6514,6 +6516,7 @@ def setUpTestData(cls):
     def setUp(self):
         self.client.force_login(self.superuser)
 
+    @ignore_warnings(category=RemovedInDjango60Warning)
```

Comment:
I think we should temporarily add `assume_scheme` to the `URLField.formfield()` instead of decorating not directly related tests with `@ignore_warnings` (`admin_views`, `admin_widgets`, `generic_inline_admin`, `model_forms`):
```diff
diff --git a/django/db/models/fields/__init__.py b/django/db/models/fields/__init__.py
index 060e1be605..feafa4aecb 100644
--- a/django/db/models/fields/__init__.py
+++ b/django/db/models/fields/__init__.py
@@ -2581,6 +2581,8 @@ class URLField(CharField):
         return super().formfield(
             **{
                 "form_class": forms.URLField,
+                # RemovedInDjango60Warning.
+                "assume_scheme": "http",
                 **kwargs,
             }
         )
```